src/Forms/DiffField.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace SilverStripe\VersionedAdmin\Forms;
+
+use LogicException;
+use SilverStripe\Forms\FormField;
+use SilverStripe\Forms\HTMLReadonlyField;
+use SilverStripe\View\Parsers\Diff;
+
+/**
+ * This form field is both a field object in it's own right, and a decorator for another field type.
+ * It is used to render differences between two data entries into the field type it wraps
+ * e.g. a TextField with the value "Old data" can be trasnformed with {@see DiffTransformation}
+ * and then having setValue called again to load in "New data" - this field will then render
+ * the value "<ins>New</ins> <del>Old</del> data". Most useful in historic version comparisons
+ * {@see SilverStripe\Versioned\Versioned}
+ */
+class DiffField extends HTMLReadonlyField
```

Comment:
This class doesn't appear to have any unit test.

Reply:
Yeah, there's no real logic in this class though. I think Behat test coverage should be sufficient instead - what do you think?